src/cli/subcommands/car_cmd.rs

```diff
@@ -0,0 +1,134 @@
+// Copyright 2019-2023 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use std::path::PathBuf;
+
+use async_recursion::async_recursion;
+use clap::Subcommand;
+use futures::{AsyncRead, StreamExt};
+use fvm_ipld_car::Block;
+use fvm_ipld_car::CarHeader;
+use fvm_ipld_car::CarReader;
+use tokio_util::compat::TokioAsyncReadCompatExt;
+
+use crate::ipld::CidHashSet;
+
+#[derive(Debug, Subcommand)]
+pub enum CarCommands {
+    Concat {
+        first: PathBuf,
```

Comment:
In general, what is preventing us from being more flexible here? That is to say, accept an arbitrary number of files, like the regular `cat` command. It should be straightforward to do with the existing code at no extra mental overhead cost.

Reply:
Yeah this is feasible, fixed